src/ogma/features/nodeGrouping.ts

```diff
@@ -97,6 +97,11 @@ export class NodeGroupingTransformation {
           style: 'bold'
         },
         layer: -1,
+        detectable: (item): boolean => {
```

Comment:
adding this because you suggested it. Feels out of scope (we can discuss if you disagree), but since it's a valuable addition (almost a bug fix, bug for another bug), I propose to keep it.